spec/bowling_spec.rb

```diff
@@ -0,0 +1,37 @@
+require_relative '../lib/bowling.rb'
+
+describe 'bowling' do
+  before{@game = Bowling.new}
+
+  it 'can roll a gutter game' do
+    20.times{@game.rollball(0)}
+    expect(@game.score).to eq(0)
```

Comment:
Nice to test a gutter game with the 20.times, I hand wrote the entry for each throw